ring_buffer/ring_buffer.py

```diff
@@ -1,9 +1,58 @@
 class RingBuffer:
+    class RingNode:
+        def __init__(self):
+            self.value = None
+            self.next = None
+
     def __init__(self, capacity):
-        pass
+        self.capacity = capacity
+        
+        # create head node
```

Comment:
Good job! Using a list for self.storage is a good idea too! 
```python
def __init__(self, capacity): 
  self.capacity = capacity
  self.storage = [] (this line can be improved)

def append(self, item):
    self.storage[self.current] = item
    self.current += 1
    #....rest of code
```
The code above is what I had in mind when I was talking to you today